lib/pages/inbox.dart

```diff
@@ -68,25 +68,30 @@ class InboxPage extends HookWidget {
       child: Scaffold(
         appBar: AppBar(
           title: RadioPicker<String>(
-            onChanged: (val) {
-              selected.value = val;
-              isc.clear();
+            title: 'account',
+            values: accStore.loggedInInstances
+                .expand(
+                  (instanceHost) => accStore
+                      .usernamesFor(instanceHost)
+                      .map((username) => '$username@$instanceHost'),
+                )
+                .toList(),
+            groupValue:
+                '${accStore.defaultUsername}@${accStore.defaultInstanceHost}',
+            onChanged: (value) {
+              final userTag = value.split('@');
```

Comment:
nit: you can actually do `final [user, instance] = value.split('@');` and it'll declare two variables.

Reply:
I copied that section directly from the user profile page, so if we change it here we should change it there too.

Reply:
This is new syntax that's only available since we updated to Dart SDK v3, so it makes sense  that the old code in user profile page doesn't have it.

Updating user profile page is also a good idea, but we don't have to do that in this PR.

Reply:
Made the requested change